src/main/rx/rx.c

```diff
@@ -113,15 +111,13 @@ static float rcRaw[MAX_SUPPORTED_RC_CHANNEL_COUNT];     // interval [1000;2000]
 float rcData[MAX_SUPPORTED_RC_CHANNEL_COUNT];           // interval [1000;2000]
 uint32_t validRxSignalTimeout[MAX_SUPPORTED_RC_CHANNEL_COUNT];
 
-#define MAX_INVALID_PULSE_TIME 300                  // hold time in millisecons after bad channel or Rx link loss
+#define MAX_INVALID_PULSE_TIME 300                  // hold time in milliseconds after bad channel or Rx link loss
+// will not be actioned until the nearest multiple of 100ms
 #define PPM_AND_PWM_SAMPLE_COUNT 3
 
-#define DELAY_50_HZ (1000000 / 50)
-#define DELAY_33_HZ (1000000 / 33)
-#define DELAY_15_HZ (1000000 / 15)
-#define DELAY_10_HZ (1000000 / 10)
-#define DELAY_5_HZ (1000000 / 5)
-#define SKIP_RC_ON_SUSPEND_PERIOD 1500000           // 1.5 second period in usec (call frequency independent)
+#define DELAY_20_MS 20000                           // 20ms
+#define DELAY_100_MS 100000                         // 100ms
+#define DELAY_1500_MS 1500000                       // 1.5 seconds
```

Comment:
`1500000` rewrite to `(1500*1000)` can improve readiblitiy and make it less error prone.

Reply:
Or better define a macro similar to `TASK_PERIOD_MS`

Reply:
done the 'easy way for me' to keep the delay in the name